.github/workflows/main.yml

```
@@ -108,10 +111,15 @@ jobs:
   test:
     name: Unit Test on Node ${{matrix.node_version}} (${{matrix.os}}) and GraphQL v${{matrix.graphql_version}}
     runs-on: ubuntu-latest
+    needs:
+      - lint
+      - prettier-check
+      - dev-tests
+      - esm
     strategy:
       matrix:
         os: [ubuntu-latest] # remove windows to speed up the tests
-        node_version: [12, 17]
+        node_version: [12, 14, 16, 18]
```

I guess you can already remove 12
```suggestion
        node_version: [14, 16, 18]
```

---

we still support `v12` I think?